backend/compact-connect/lambdas/python/common/cc_common/data_model/data_client.py

```diff
@@ -932,6 +931,45 @@ def process_registration_values(
                 ]
             )
 
+    @paginated_query
+    @logger_inject_kwargs(logger, 'compact', 'provider_id', 'detail', 'jurisdiction', 'license_type')
+    def get_privilege_data(
+        self,
+        *,
+        compact: str,
+        provider_id: str,
+        detail: bool = False,
+        dynamo_pagination: dict,
+        jurisdiction: str,
+        license_type_abbr: str,
+        consistent_read: bool = False,
+    ):
```

Comment:
Please add a return type annotation here 
```suggestion
    ) -> dict:
```